src/platform/Tizen/AppPreference.cpp

```diff
@@ -16,10 +16,22 @@
  */
 
 #include "AppPreference.h"
+
 #include <app_preference.h>
```

Comment:
When correcting includes, can you group them according to Google codestyle: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes I'm not sure whether there is a codestyle for includes in connectedhomeip project, but I think that we can adopt Google codestyle for that:

1. *dir2/foo2.h*.
2. A blank line
3. C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
4. A blank line
5. C++ standard library headers (without file extension), e.g., <algorithm>, <cstddef>.
6. A blank line
7. Other libraries' .h files (e.g. Tizen Native API headers, GLIB headers, etc.).
8. A blank line
8. Your project's .h files (matter includes).